css-color/Overview.bs

```diff
@@ -161,6 +160,75 @@ Notes On Using Colors</h3>
 		Really?  Isn't HD video in Rec.709? Shouldn't video be consistent with images?
 		Or do implementations really do this differently?
 
+<h2 id='resolving-color-values'>
+Resolving Color values</h2>
+
+	Various properties accept <<color>> values.
+	Unless an exception is explicitly defined,
+	all such properties must <dfn lt="resolve color values|resolving color values">resolve color values</dfn> as defined below
+	to determine the <a>computed value</a> and the <a>used value</a> for <<color>>.
+
+	<ul>
+		<li>
+		The computed value of the ''<color>/currentcolor'' keyword is ''<color>/currentcolor''.
+
+		If ''<color>/currentcolor'' is used as the value of the 'color' property,
+		the used value is the same as if ''inherit'' had been specified instead.
+		For all other properties that accept a <<color>>,
+		the used value of ''<color>/currentcolor'' is the same as
+		the used value of the 'color' property on the same element.
+
+		Note: This means that if the ''<color>/currentcolor'' value is inherited,
+		it’s inherited as a keyword,
+		not as the value of the 'color' property,
+		so descendants will use their own 'color' property to resolve it.
+
+		<li>
+		The computed value and used value of ''<color>/transparent'' is ''rgba(0,0,0,0)''.
+
+		<li>
+		The computed value and used value of
+		<a>named colors</a> (including <<deprecated-system-color>> colors),
+		3 and 6 digits <a>hex colors</a>,
+		comma separated ''rgb()'' colors without an alpha channel,
+		and comma separated ''hsl()'' colors without an alpha channel,
+		is the equivalent numeric value in comma separated ''rgb()'' notation omitting the alpha value.
+
+		<li>
+		The computed value and used value of
+		4 and 8 digits <a>hex colors</a>,
+		comma separated ''rgba()'' colors with an alpha channel,
+		and comma separated ''hsla()'' colors with an alpha channel,
+		is the equivalent numeric value in comma separated ''rgba()'' notation with the alpha value.
```

Comment:
This doesn't match what browsers currently do. Browsers use `rgb()` for opaque colors even if the color is specified from `rgba()`. See http://software.hixie.ch/utilities/js/live-dom-viewer/saved/4780